### ISSUE TITLE
Istio 1.4.0 tracing service moved from port 80 to 9411

### ISSUE THREAD

**Issue by Stono** [istio/istio#19227]
**Bug description**
In istio 1.4.0, `tracing.istio-system` has moved from port `80` to `9411`.  There is already a `zipkin.istio-system` service in the namespace on `9411`.  Both services point to the same workload (Jaeger).

This is a breaking change for us so just wondering why it was made and if it was intentional, it should have been in the upgrade notes (as we have upstream systems talking to tracing.istio-system).  

**Expected behavior**
Consistent service ports between releases or details in release notes explaining the changes.

OR if you're really honouring semver releases, 1.4.0 shouldn't have breaking changes from 1.3.x and a different service port could probably be classed as breaking.

**Steps to reproduce the bug**

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**
1.4.0

**How was Istio installed?**

**Environment where bug was observed (cloud vendor, OS, etc)**

**Comment by Stono** [comment]
Associated commit appears to be https://github.com/istio/istio/commit/9e0887f5cef4ef4038f464e113be28d02937c782#diff-264788198be8f0b9371c31fecce6d199 and issue https://github.com/istio/istio/issues/17204

**Comment by sdake** [comment]
Nodeport more trouble then it is worth...  These two PRs need a revert.  The author is free to resubmit the work.

**Comment by sdake** [comment]
Had a deeper look at the PR.  With `istioctl dashboard jeager` for example, none of these config options are needed across any of the third party component charts.  We should consider their removal.

@Stono for the moment, I am going to make a small surgical change that should exhibit the original behavior.

**Comment by Stono** [comment]
Thanks @sdake - no huge urgency on my part, I've patched around it locally.  Just don't want other users getting caught out when they upgrade to `1.4.0`.

**Comment by sdake** [comment]
@Stono this was fixed by an unrelated commit that syncronized the operator with the installer charts after my PR to fix the installer charts had merged for 1.4.

Cheers
-steve

**Comment by sdake** [comment]
verified 1.4-dev (what will be 1.4.1) operator instantiates the services as follows

```
...
tracing                  ClusterIP      10.111.253.18    <none>          80/TCP                                                                                                                       13s
zipkin                   ClusterIP      10.97.243.159    <none>          9411/TCP                                                                                                                     14s
```

Cheers
-steve
